data-api/schemas/crm4.yml

```diff
@@ -103,7 +101,7 @@ Crm4CaseDetailsDTO:
       $ref: '#/Crm4ClientDTO'
     ProceedingDetails:
       $ref: '#/Crm4ProceedingDetails'
```

Comment:
This should be named `Crm4ProceedingDetailsDTO` as the rest

Reply:
Done.Thanks